tokens.xml

```diff
@@ -1387,16 +1387,25 @@ This creature can block only creatures with flying.</text>
                 <pt>1/1</pt>
             </prop>
             <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/3/f/3f03e9fe-cb71-498e-bf16-5af6af37a192.jpg?1650820069">SNC</set>
+            <reverse-related count="x">Boss's Chauffeur</reverse-related>
             <reverse-related count="2">Cabaretti Charm</reverse-related>
+            <reverse-related count="2">Caladaia Guardian</reverse-related>
             <reverse-related attach="attach">Citizen's Crowbar</reverse-related>
             <reverse-related>Courier's Briefcase</reverse-related>
             <reverse-related>Darling of the Masses</reverse-related>
             <reverse-related>Exhibition Magician</reverse-related>
+            <reverse-related count="x">Grand Crescendo</reverse-related>
             <reverse-related>Halo Fountain</reverse-related>
+            <reverse-related count="x">Indulge // Excess</reverse-related>
+            <reverse-related>Kitt Kanto, Mayhem Diva</reverse-related>
             <reverse-related>Luxurious Libation</reverse-related>
+            <reverse-related count="x">Master of Ceremonies</reverse-related>
```

Comment:
should this be an x? the other player will just make one token

Reply:
The player that controls the Master of Ceremonies could create multiple tokens, as they create 1 for each opponent that chooses that specific mode. Could maybe make it 1 by default and count=x as an excluded.

Reply:
I think that would be more intuitive yeah, for fast gameplay your opponents can simply make their own tokens and you copy them

Reply:
```suggestion
            <reverse-related>Master of Ceremonies</reverse-related>
            <reverse-related count="x" exclude="exclude">Master of Ceremonies</reverse-related>
```